src/Lorentz/Contracts/TZBTC/Impl.hs

```diff
@@ -29,17 +29,15 @@ module Lorentz.Contracts.TZBTC.Impl
   , unpause
   ) where
 
-import Prelude hiding (drop, get, some, swap, (>>))
+import Prelude ()
```

Comment:
Is it needed at all? If the intent is to import instances, I am pretty sure they are brought into scope by some other import. If the intent is to disable the implicit import of `Prelude`, I suppose it's already implied by `RebindableSyntax` (but I am not 100% sure).

Reply:
Loool, I didn't consider the option that this import was useless from the very beginning,